api/v1alpha1/dataprotectionapplication_types.go

```diff
@@ -448,9 +454,16 @@ type NonAdmin struct {
 	EnforceRestoreSpec *velero.RestoreSpec `json:"enforceRestoreSpec,omitempty"`
 
 	// GarbageCollectionPeriod defines how frequently to look for possible leftover non admin related objects in OADP namespace.
+	// A value of 0 disables garbage collection.
 	// By default 24h
 	// +optional
 	GarbageCollectionPeriod *metav1.Duration `json:"garbageCollectionPeriod,omitempty"`
+
+	// BackupSyncPeriod defines how frequently to sync backups in OADP namespace to non admin namespaces.
```

Comment:
NIT: specifies the interval at which backups **from** the OADP namespace are synchronized with non-admin namespaces.

Reply:
done